pkg/webhooks/admission/jobs/validate/admit_job.go

```diff
@@ -375,22 +375,5 @@ func validateTaskTopoPolicy(task v1alpha1.TaskSpec, index int) string {
 		return fmt.Sprintf("spec.task[%d] isn't Guaranteed pod, kind=%v", index, v1qos.GetPodQOS(pod))
 	}
 
-	for id, container := range append(template.Spec.Containers, template.Spec.InitContainers...) {
-		requestNum := guaranteedCPUs(container)
-		if requestNum == 0 {
-			return fmt.Sprintf("the cpu request isn't  an integer in spec.task[%d] container[%d].",
-				index, id)
-		}
-	}
-
 	return ""
 }
-
-func guaranteedCPUs(container v1.Container) int {
```

Comment:
/cc @huone1 I tried to track the original commit. It seems that this validation comes from CPU topology implementaion. Please help take a look at the commit.